core/src/controllers/rows.test.js

```diff
@@ -69,6 +69,18 @@ describe('Rows Endpoints', () => {
     expect(res.body.data[0].lastName).not.toBeNull();
   });
 
+  it('GET /tables/:name/rows: should filter items by createdAt date', async () => {
+    const res = await requestWithSupertest.get(
+      '/api/tables/users/rows?_filters=createdAt__gte:2009-01-01 00:00:00'
+    );
+
```

Comment:
thanks @AbegaM - could we also do a more pessimistic test too, please?

So:
* ask for `gte` for a date in 2010
* a `lte` for a date in 2008
* plus `_eq` and `neq` for specific dates? (test `__eq` for a date that is present, and a date that isn't present)